<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>518</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>43</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
 background-color: #2e2f30;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="currentTrack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel
{

    color: white; 
border:2px solid rgb(255,255,255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>  НАЗВАНИЕ ПЕСНИ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Seek_forward">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
height:10px;
width:460px;
background: purple;
border-radius:5px;
}
QSlider::handle:horizontal
{
background:rgb(52,50,64);
width:20px;
height:20px;
margin: -7px -7px;
border-radius: 100px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
height:10px;
width:460px;
background: purple;
border-radius:5px;
}
QSlider::handle:horizontal
{
background:rgb(52,50,64);
width:20px;
height:20px;
margin: -7px -7px;
border-radius: 100px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Play_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
height:10px;
width:460px;
background: purple;
border-radius:5px;
}
QSlider::handle:horizontal
{
background:rgb(52,50,64);
width:20px;
height:20px;
margin: -7px -7px;
border-radius: 100px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pause">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
height:10px;
width:460px;
background: purple;
border-radius:5px;
}
QSlider::handle:horizontal
{
background:rgb(52,50,64);
width:20px;
height:20px;
margin: -7px -7px;
border-radius: 100px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Seek_Back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
height:10px;
width:460px;
background: purple;
border-radius:5px;
}
QSlider::handle:horizontal
{
background:rgb(52,50,64);
width:20px;
height:20px;
margin: -7px -7px;
border-radius: 100px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Volume">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>328</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal
{
height: 10px;
width: 380px;
background:purple;
border-radius:5px;
}

QSlider::handle:horizontal
{
background:rgb(54,50,64);
width:20px;
height:20px;
margin:20px 12px;
border-radius:10px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Volume">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
border-radius:40px;
background-color: purple;
color:rgb(52,51,64);
padding:0px
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QTableView" name="playlistView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:white;
background color: #444444;

    color: white; </string>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>411</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
height: 10px;
    width: 460px;
    background: #2e2f30;
    color: white; }

QSlider::handle:horizontal
{
background: white;
font:white;
width:20px;
height:20px;

}
</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionOpen_Audio_File">
   <property name="text">
    <string>Open Audio File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
